lib/widgets/product_listing.dart

```diff
@@ -517,3 +522,22 @@ class _ProductListingState extends State<ProductListing> {
     }
   }
 }
+
+  StatelessWidget _getProductStockQuantityTag(
+    Product product
```

Comment:
For the formatting practices, we use VS Code's native formatting option: https://stackoverflow.com/a/29973358
On Mac: `Shift` + `Option` + `F`
Please use the formatting option to format the entire document.